jdaviz/configs/cubeviz/plugins/cube_listener.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+from contextlib import contextmanager
+import sys
+import os
+import time
+
+try:
+    from strauss.sonification import Sonification
+    from strauss.sources import Events
+    from strauss.score import Score
+    from strauss.generator import Spectralizer
+except ImportError:
+    pass
+
+MINVOL = 1/(2**15 - 1)
```

Comment:
Please add comment to explain math here.

Reply:
```suggestion
#  Minimum non-zero fraction of the total signal amplitude that can be represented by a 16-bit signed integer
MINVOL = 1/(2**15 - 1)
```

Reply:
@javerbukh were you going to commit this suggestion?

Reply:
Added it in a separate commit, this can be resolved.